setup.cfg

```diff
@@ -1,74 +0,0 @@
-[metadata]
-name = Pillow
-description = Python Imaging Library (Fork)
-long_description = file: README.md
-long_description_content_type = text/markdown
-url = https://python-pillow.org
-author = Jeffrey A. Clark (Alex)
-author_email = aclark@aclark.net
-license = HPND
-classifiers =
-    Development Status :: 6 - Mature
-    License :: OSI Approved :: Historical Permission Notice and Disclaimer (HPND)
-    Programming Language :: Python :: 3
```

Comment:
Why is "Programming Language :: Python :: 3" not present in pyproject.toml?

Reply:
pyproject-fmt removes it:
https://github.com/tox-dev/pyproject-fmt/issues/106